cms/djangoapps/contentstore/views/videos.py

```diff
@@ -305,6 +311,11 @@ def videos_post(course, request):
             for file in request.json["files"]
     ):
         error = "Request 'files' entry does not contain 'file_name' and 'content_type'"
+    elif any(
+            file['content_type'] not in VIDEO_SUPPORTED_FILE_FORMATS.values()
+            for file in request.json["files"]
+    ):
+        error = "Request 'files' entry contain unsupported content_type"
```

Comment:
@muzaffaryousaf I have made this because messages here backend are like that, and normal user would not get this, unless he sends some ajax through console or any other way. User facing message comes from JS-side.